blackjack.py

```diff
@@ -104,17 +113,16 @@ def get_hard_hand_value(self):
         :return: int
         """
         hard_hand_value = 0
-        for card in self.hand:
-            if self.hand[0].value == 'Ace' and self.hand[1].value in ['Jack', 'Queen', 'King']:
-                hard_hand_value = 21.1
-            elif self.hand[0].value in ['Jack', 'Queen', 'King'] and self.hand[1].value == 'Ace':
-                hard_hand_value = 21.1
-            elif card.value.isdigit():
-                hard_hand_value += int(card.value)
-            elif card.value == 'Ace':
-                hard_hand_value += 11
-            else:
-                hard_hand_value += 10
+        if self.check_natural_win():
+            hard_hand_value = 21.1
+        else:
+            for card in self.hand:
+                if card.value.isdigit():
+                    hard_hand_value += int(card.value)
+                elif card.value == 'Ace':
+                    hard_hand_value += 11
+                else:
+                    hard_hand_value += 10
```

Comment:
Because this method was almost a perfect copy/paste of another one, that should trigger a warning that you may be able to factorize your code.